windows_containers/windows-containers-release-notes-6-x.adoc

```diff
@@ -31,23 +31,35 @@ endif::openshift-origin[]
 [id="wmco-6-0-0"]
 == Release notes for Red Hat Windows Machine Config Operator 6.0.0
 
-This release of the WMCO provides bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 6.0.0 were released in
+This release of the WMCO provides bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 6.0.0 were released in link:https://access.redhat.com/errata/RHBA-2022:97930[RHBA-2022:97930].
```

Comment:
(We have discussed in Slack but noting the Live ID issue here)

Reply:
so now we have a live ID :)